manifests/f/Facebook/Messenger/202.0.552992785/Facebook.Messenger.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Facebook.Messenger
+PackageVersion: 202.0.552992785
+InstallerLocale: en-US
+InstallerType: nullsoft
+UpgradeBehavior: uninstallPrevious
+ProductCode: c1b3adcf-2068-5e8d-b25d-30ce588e3a4c
+ReleaseDate: 2024-01-04
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+Installers:
+- Architecture: x64
+  InstallerUrl: https://www.messenger.com/messenger/desktop/downloadV2/?platform=win
+  InstallerSha256: 0DB6164EA22EE1F0685EDF619D369D97D5FDA55B93201210A597A8E78402E162

```

Comment:
```suggestion
  InstallerSha256: CE04BB82D4C0488351B516F86990AA6F5DC4CAFF675AC157FE78B91B68435CCE
```

Reply:
Strange. Because this is what I got:
![image](https://github.com/microsoft/winget-pkgs/assets/5486625/57494fb4-cacc-490f-8512-843bfda09164)

Reply:
Does this package's hash frequently change? We have another package which has a different version number and hash based on the user's IP address and the time of day. Is it possible that a similar situation is occurring here?

Reply:
It would only change when there is a new version. But the version in this PR has been released for a few days.